package.json

```diff
@@ -73,6 +73,11 @@
       "svgo --disable=removeViewBox --pretty"
     ]
   },
+  "prettier": {
```

Comment:
I'm not sure we should add new Prettier options as part of a PR. Probably best to do this separately and get the team to agree.

Reply:
An unintentional change :grimacing: